benefits.md

```diff
@@ -99,15 +99,16 @@ Base Two provides 12 paid holidays in which the Base Two office will be closed.
 1. New Year's Day (January 1st)
 2. Martin Luther King Day (3rd Monday in January)
 3. Presidents' Day (3rd Monday in February)
-4. Earth Day (April 22nd or nearest work day)
+4. Earth Day (April 22nd)
 5. Memorial Day (last Monday in May)
-6. Independence Day (4th of July)
-7. Labor Day (1st Monday in September)
-8. Veterans Day (November 11th)
-9. Thanksgiving (4th Thursday in November)
-10. Consumerism Day (day after Thanksgiving)
-11. Christmas Eve (December 24th)
-12. Christmas Day (December 25th)
+6. Juneteenth (June 19th)
```

Comment:
💬 We'll need to update the website and Base Two calendar too.

Reply:
Calendar (Base Two Celebrations) has been updated.